django/contrib/contenttypes/checks.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from itertools import chain
+
 from django.apps import apps
 from django.utils import six
 
 
-def check_generic_foreign_keys(**kwargs):
+def check_generic_foreign_keys(app_configs=None, **kwargs):
```

Comment:
Adjusted this method to correctly deal with specified `app_configs`. I guess this could be part of another commit.